podpac/core/coordinates/coordinates.py

```diff
@@ -801,6 +855,55 @@ def hash(self):
         json_d = json.dumps(self.full_definition, separators=(",", ":"), cls=podpac.core.utils.JSONEncoder)
         return hash_alg(json_d.encode("utf-8")).hexdigest()
 
+    @property
+    def geotransform(self):
+        """ :tuple: GDAL geotransform. """
+        # Make sure we only have 1 time and alt dimension
+        if "time" in self.udims and self.ushape[self.udims.index("time")] > 1:
+            raise TypeError(
+                'Only 2-D coordinates have a GDAL transform. This array has a "time" dimension of {} > 1'.format(
+                    self.ushape[self.udims.index("time")]
+                )
+            )
+        if "alt" in self.udims and self.ushape[self.udims.index("alt")] > 1:
+            raise TypeError(
+                'Only 2-D coordinates have a GDAL transform. This array has a "alt" dimension of {} > 1'.format(
+                    self.ushape[self.udims.index("alt")]
+                )
+            )
+
+        # Do the uniform coordinates case
+        if (
+            "lat" in self.dims
+            and isinstance(self._coords["lat"], UniformCoordinates1d)
+            and "lon" in self.dims
+            and isinstance(self._coords["lon"], UniformCoordinates1d)
+        ):
+            if self.dims.index("lon") < self.dims.index("lat"):
+                first, second = "lat", "lon"
+            else:
+                first, second = "lon", "lat"  # This case will have the exact correct geotransform
+            transform = rasterio.transform.Affine.translation(
+                self[first].start - self[first].step / 2, self[second].start - self[second].step / 2
+            ) * rasterio.transform.Affine.scale(self[first].step, self[second].step)
+            transform = transform.to_gdal()
+        elif (
+            "lat,lon" in self.dims
+            or "lon,lat" in self.dims
+            and isinstance(self._coords.get("lat,lon", self._coords.get("lon,lat")), RotatedCoordinates)
+        ):
+            transform = self._coords.get("lat,lon", self._coords.get("lon,lat")).geotransform

```

Comment:
You can ignore me on this one, but I scratched my head for a few moments. It would much easier to read (and actually fewer lines) as two cases. This still works, right?

```
        elif "lat,lon" in self.dims and isinstance(self._coords["lat,lon"], RotatedCoordinates):
            transform = self._coords["lat,lon"].geotransform
        elif "lon,lat" in self.dims and isinstance(self._coords["lon,lat"], RotatedCoordinates):
            transform = self._coords['lon,lat'].geotransform
```

Reply:
much better.